LSPS0/README.md

```diff
@@ -240,8 +240,8 @@ Other LSPS specifications:
   the key names of the `params` for each `method`, and the
   meanings of each parameter.
   * MUST prefix `method` names with `lsps` followed by the LSPS
-    number followed by `.`, e.g.  `lsps999.dothis` for a client
-    request or `lsps999.thathappened` for an LSP notification.
+    number followed by `.`, e.g.  `lsps999.do_this` for a client
+    request or `lsps999.that_happened` for an LSP notification.
```

Comment:
Might as well add the snake case rule here

Reply:
Added a bullet for this.